src/app/website/company-home/feature-summary/feature-summary.component.html

```diff
@@ -0,0 +1,29 @@
+<section class="featuresummary">
+  <div class="row featuresummary__content"
+       *ngFor="let data of contentData">
+    <div class="row space"></div>
+    <div class="row">
+      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 img-column text-center">
+        <img src={{data?.assetValue.fields.file.url}}
+             alt="image"
+             class="featuresummary__content-img img-responsive">
+      </div>
+      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 text-column text-center">
```

Comment:
you have to talk to Manikya about bootstrap error here. Also in line 6 and whereever you have used it should change